corgi/tasks/brew.py

```diff
@@ -235,11 +237,11 @@ def save_component(
         name=meta.pop("name", ""),
         version=component_version,
         release=meta.pop("release", ""),
-        arch=meta.pop("arch", ""),
+        arch=meta.pop("arch", "noarch"),
         defaults={
             "description": meta.pop("description", ""),
             "filename": find_package_file_name(meta.pop("source_files", [])),
-            "namespace": component.get("namespace", ""),
+            "namespace": Component.Namespace.REDHAT if rpm else Component.Namespace.UPSTREAM,
```

Comment:
Note to self: This "rpm" kwarg should go away - we can just do below instead:
```suggestion
            "namespace": Component.Namespace.REDHAT if component_type = Component.Type.RPM.upper() else Component.Namespace.UPSTREAM,
```